networks/rpc/websocket.go

```diff
@@ -96,6 +96,8 @@ func (srv *Server) FastWebsocketHandler(ctx *fasthttp.RequestCtx) {
 	}
 
 	err := upgrader.Upgrade(ctx, func(conn *fastws.Conn) {
+		conn.SetReadDeadline(time.Now().Add(5 * time.Minute)) // Each connection will be closed after the deadline
```

Comment:
How about using a config or at least declaring a const for this value?

Reply:
Thanks for the comment. This PR is only for a hotfix for public EN. It will not be merged in dev branch.